testing/run-scripts/image_make.sh

```diff
@@ -13,56 +13,6 @@ source "${BASH_SOURCE%/*}/utils.sh" || (echo "cannot find utils.sh" && exit 1)
 BROWSER=$1
 VERSION=$2
 
-case $BROWSER in
-    chr|chrome|CHROME)
-        BROWSER=chrome
-        case $VERSION in
-            dev|DEV) VERSION=dev ;;
-            rel|release|REL|RELEASE) VERSION=rel ;;
-            canary|CANARY) VERSION=canary ;;
-            *)
-                log "Unknown version of chrome: $2.   Options are dev, rel(ease), and canary."
-                exit 1;
-                ;;
-        esac
-        ;;
-    lcr|lchrome|localchrome|LCR|LCHROME|LOCALCHROME)
-        BROWSER=localchrome
-        case $VERSION in
-            debug|dbg|d|DEBUG|DBG|D|Debug)
-                VERSION=debug
-                # validate the path.
-                chrome_build_path Debug
-                ;;
-            release|rel|r|RELEASE|REL|R|Release)
-                VERSION=release
-                # validate the path.
-                chrome_build_path Release
-                ;;
-            *)
-                log "Unknown version of localchrome: $2. Options are debug and release."
```

Comment:
We still need to validate the build (release/debug) of localchrome. Perhaps in get_localchrome above?